docs/content/guides/author-apps/dapr/how-to-dapr-building-block/index.md

```diff
@@ -0,0 +1,103 @@
+---
+type: docs
+title: "How-To: Add a Dapr building block"
+linkTitle: "Add a Dapr building block"
```

Comment:
If this wraps around dur to length you can remove the "Dapr" from the linkTitle.